packages/react-components/src/components/ChatMessage/ChatMessageContent.tsx

```diff
@@ -187,12 +187,11 @@ const extractContentForAllyMessage = (props: ChatMessageContentProps): string =>
     });
 
     // Inject attachment names for aria.
-    if (attachments) {
-      let attachmentList = '';
-      /* @conditional-compile-remove(attachment-download) */
-      attachmentList = attachmentCardGroupDescription(props);
+    /* @conditional-compile-remove(attachment-download) */
```

Comment:
Small behavioural change with the cc moving outside of the `if` condition but this should be ok since the previous logic should have been around all attachments list anyways. But if anyone sees a reason why this might break stable builds please bring it up

Reply:
this if condition was added by me few weeks ago specifically for file sharing: https://github.com/Azure/communication-ui-library/pull/4438/files


added comments to explain this a bit